scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/MissingAssociationConflict.java

```diff
@@ -0,0 +1,28 @@
+package gov.nasa.jpl.aerie.scheduler;
+
+import gov.nasa.jpl.aerie.constraints.time.Windows;
+
+import java.util.Collection;
+
+public class MissingAssociationConflict extends Conflict {
+  private final Collection<ActivityInstance> instances;
+
+  /**
+   * ctor creates a new conflict
+   *
+   * @param goal IN STORED the dissatisfied goal that issued the conflict
+   */
+  public MissingAssociationConflict(final Goal goal, final Collection<ActivityInstance> instancesToChooseFrom) {
+    super(goal);
+    this.instances = instancesToChooseFrom;
+  }
```

Comment:
This conflict says that there is a missing association. And sometimes you can associate to several instances (because several corresponds) and sometimes only one. and the decision is made by the scheduler, not the conflict itself.